cranelift/codegen/src/isa/riscv64/inst/mod.rs

```diff
@@ -1459,21 +1456,26 @@ impl Inst {
                 let mut s = "args".to_string();
                 let mut empty_allocs = AllocationConsumer::default();
                 for arg in args {
-                    use std::fmt::Write;
                     let preg = format_reg(arg.preg, &mut empty_allocs);
                     let def = format_reg(arg.vreg.to_reg(), allocs);
                     write!(&mut s, " {}={}", def, preg).unwrap();
                 }
                 s
             }
-            &Inst::Ret { ref rets } => {
+            &Inst::Ret {
+                ref rets,
+                stack_bytes_to_pop,
+            } => {
                 let mut s = "ret".to_string();
+                if stack_bytes_to_pop != 0 {
+                    write!(&mut s, " {stack_bytes_to_pop}").unwrap();
```

Comment:
Rather than print a pseudo-inst here, could we print e.g. `add sp, sp, #imm ; ret" or similar?